src/annotator/guest.js

```diff
@@ -34,28 +47,20 @@ const animationPromise = fn =>
 function annotationsForSelection() {
   const selection = /** @type {Selection} */ (window.getSelection());
   const range = selection.getRangeAt(0);
-  return rangeUtil.itemsForRange(range, node => $(node).data('annotation'));
+  return rangeUtil.itemsForRange(
+    range,
+    node => /** @type {AnnotationHighlight} */ (node)._annotation
+  );
 }
 
 /**
  * Return the annotations associated with any highlights that contain a given
  * DOM node.
```

Comment:
The `@return` type here can be inferred by TS, obviously, from the `@type` below, but do we have a convention vis-a-vis readability for devs? This is somewhat of a poor example, this function, as it's so short, but could return type be more buried in longer functions? Just ruminating...

Reply:
> The @return type here can be inferred by TS, obviously, from the @type below, but do we have a convention vis-a-vis readability for devs? 

We don't have clear conventions yet. I think devs should feel free to ask for it to be added in reviews if they think it will make the code easier to follow.